STYLEGUIDE.md

```diff
@@ -0,0 +1,172 @@
+# Style guide and design principles
+
+This guide provides a set of design principles and guidelines for the `yoneda`
+project. Our style and design principles borrows heavily from
+[`agda-unimath`](https://github.com/UniMath/agda-unimath).
+
+## The structure of code
+
+We enforce strict formatting rules. This formatting allows the type of the
+defined term to be easily readable, and aids in understanding the structure of
+the definition.
+
+The general format of a definition is as follows:
+
+```rzk
+#def concat
+  ( p : x = y)
+  ( q : y = z)
+  : (x = z)
+  := idJ (A , y , \ z' q' -> (x = z') , p , z , q)
+```
+
+- We start with the name, and place every assumption on a new line.
```

Comment:
We also need to specify where to put `uses (...)`. Currently we put in on the same line as the identifier being defined.

Reply:
Agreed